StreetCare/StreetCare/Views/Screens/ICanHelpView.swift

```diff
@@ -0,0 +1,86 @@
+        VStack(alignment: .leading, spacing: 20) {
+                    HStack {
+                        Spacer()
+                        Button(action: { isPresented = false }) {
+                            Image(systemName: "xmark")
+                                .foregroundColor(.gray)
+                        }
+                    }
+                    
+                    Text("Make sure you are not going alone")
+                        .font(.headline)
+                    
+                    Text("Group presence offers security and effectiveness in engaging with unfamiliar situations and individuals, benefiting both volunteers and the homeless.")
+                        .font(.subheadline)
+                        .foregroundColor(.secondary)
+                    
+                    Text("How outreach on Street Care works?")
+                        .font(.subheadline)
+                        .fontWeight(.semibold)
+                    
+                    Text("We post the outreach for you and other volunteers can sign up to go with you.")
+                        .font(.subheadline)
+                        .foregroundColor(.secondary)
+                    
+                    Button(action: {
+                        // Action for RSVP EXISTING OUTREACH
+                    }) {
+                        Text("RSVP EXISTING OUTREACH")
+                            .frame(maxWidth: .infinity)
+                            .padding()
+                            .background(Color(#colorLiteral(red: 0.208, green: 0.290, blue: 0.129, alpha: 1)))
+                            .foregroundColor(.yellow)
+                            .cornerRadius(8)
+                    }
+                    
+                    Button(action: {
+                        isPresented = false
+                    }) {
+                        Text("CANCEL")
+                            .frame(maxWidth: .infinity)
+                            .padding()
+                            .background(Color(#colorLiteral(red: 0.208, green: 0.290, blue: 0.129, alpha: 1)))
+                            .foregroundColor(.yellow)
+                            .cornerRadius(8)
+                    }
+                    
+                    Button(action: {
+                        isOutreachCreated = true
+                    }) {
+                        Text("CREATE AN OUTREACH")
+                            .frame(maxWidth: .infinity)
+                            .padding()
+                            .background(Color(#colorLiteral(red: 0.208, green: 0.290, blue: 0.129, alpha: 1)))
+                            .foregroundColor(.yellow)
+                            .cornerRadius(8)
+                    }
+                }
+                .padding()
+                .frame(maxWidth: 300)
+                .background(Color.white)
+                .cornerRadius(12)
+                .shadow(radius: 10)
+                .alert(isPresented: $isOutreachCreated) {
```

Comment:
This is can be confusing for another developer picking up this feature as the code does not create an event yet